LondonBritishLibrary/orient/BLorient5426.xml

```diff
@@ -0,0 +1,193 @@
+                                    <incipit xml:lang="gez">በስመ፡<gap reason="ellipsis"/> ጸሎት፡ መንገደ፡ ሰማይ፡ ዕቀበኒ፡ ክርስቶስ፡ ከመ፡ ኢያእቅፍዋ፡ ለነፍስየ፡ መላእክተ፡ 
+                                        ጽልመት፡ ፈኑ፡ ሊተ፡ ሚካኤል፡</incipit>
+                                    <textLang mainLang="gez"/>                            
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.4">
+                                    <title>Prayer 4</title>
+                                    <incipit xml:lang="gez">በስመ፡<gap reason="ellipsis"/> ጸሎተ፡ መንገደ፡ ሰማይ፡ እቀበኒ፡ ክርስቶስ፡ ከመ፡ ኢያእቅፍዎ፡ መላእክተ፡ ጽልመት፡
+                                        ዘሐነፀ፡ ጽርኃ፡ ኀበ፡ አልቦ፡ ዘየአምሮ፡</incipit>
+                                    <textLang mainLang="gez"/>                            
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.5">
+                                    <title>Prayer 5</title>
+                                    <incipit xml:lang="gez">በስመ፡<gap reason="ellipsis"/> ጸሎተ፡ መንገደ፡ ሰማይ፡ አሜሃ፡ 
+                                        በደኃ<supplied reason="undefined" resp="PRS8999Strelcyn">ሪ፡</supplied> መዋዕል፡</incipit>
+                                    <textLang mainLang="gez"/>                            
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.6">
+                                    <title>Prayer 6</title>
+                                    <incipit xml:lang="gez">በስመ፡<gap reason="ellipsis"/> ጸሎተ፡ መንገደ፡ ሰማይ፡ ዕቀበኒ፡ ክርስቶስ፡ ከመ፡ ኢያእቅፍዋ፡ ለነፍስየ፡ መላእክተ፡
+                                        ጽልመት፡ ወሶቤሃ፡ ነገሮሙ፡ ለ፲ወ፪ሐዋርያት፡</incipit>
+                                    <textLang mainLang="gez"/>                            
+                                </msItem>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT2765RepCh49"/>
+                            <incipit xml:lang="gez">ሰላም፡ ለከ፡ ሰዳዴ፡ ሰይጣናት፡ ፋኑኤል፡ ለእ<hi rend="ligature">ግዚ</hi>አብሔር፡ 
+                                እምጽ<supplied reason="undefined" resp="PRS8999Strelcyn">ር</supplied>ሁ፡ ከመ፡ ኢይስክዩ፡ ሰብዓ፡ ወኢይኔሥሁ፡ ለጌጋይ፡ 
+                                ዘዘዚአሁ።</incipit>
+                            <textLang mainLang="gez"/>                            
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT1758Lefafa" type="incomplete">Formula of benediction given to the Archangel Michael</title>
+                            <note>Developed formula of benediction given by God to the Archangel Michael for the scribe, the owner, etc. of this
+                                manuscript. The end is missing.</note>
+                            <incipit xml:lang="gez">ወይቤሎ፡ ሚካኤል፡ ለእ<hi rend="ligature">ግዚ</hi>አብሔር፡ ምንትኑ፡ ድምጸ፡ ነጐድጓድ፡ ዘሰማእኩ፡ ኢይለክፍዎ፡ 
```

Comment:
Here it seems like a large ellipsis has been made, compared to the text given by Strelcyn. Maybe this should be marked?

Reply:
I will add the missing parts of what was given by Strelcyn.